src/main/java/com/serhiiklymchuk/gmailpet/config/WebConfig.java

```diff
@@ -0,0 +1,13 @@
+package com.serhiiklymchuk.gmailpet.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebConfig implements WebMvcConfigurer {
+    @Override
+    public void addViewControllers(ViewControllerRegistry registry) {
+        registry.addRedirectViewController("/", "/login");
```

Comment:
this should redirect to inbox, not to login.
Inbox, not implemented right now, but it's ok, as long as it will be implemented in the next PR